addons/website/data/website_data.xml

```diff
@@ -25,15 +25,16 @@
                     <t t-call="website.layout">
                         <div id="wrap">
                             <div class="oe_structure">
-                                <section class="s_title pt24 pb24" data-vcss="001" data-snippet="s_title">
+                                <section class="s_title pt24 pb24 o_colored_level o_cc o_cc4 oe_img_bg" data-vcss="001"
```

Comment:
Can you use an existing picture (that will be replaced by theme (most of the time) e.g. s_banner in a parallax with class black-50 to have your dark picture, but avoid a new hardcoded 24Ko picture.

Not sure that after you refactoring of parralax it will be still possible, but meanwhile, seems the better solution imo, instead to add a new picture.

Reply:
I made the change. you're right, it's much better like that (the picture that change with the theme)